extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -89,7 +89,9 @@ public function executePicker() {
 	}
 
 	public function executePreview() {
+		global $wgEnablePortableInfoboxEuropaTheme;
 
+		$this->infoboxTheme = $wgEnablePortableInfoboxEuropaTheme ? 'theme="europa"' : '';
```

Comment:
If you set ``theme="europa"`` in infobox markup this part won't be needed. What do you think about that?